### PR TITLE
Replace klog.Infof with klog.Info when no var

### PR DESCRIPTION
**Pull request by mysunshine92**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
 
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kubelet/app/server.go [hunk 1]
```diff
@@ -476,7 +476,7 @@ func makeEventRecorder(kubeDeps *kubelet.Dependencies, nodeName types.NodeName)
 	kubeDeps.Recorder = eventBroadcaster.NewRecorder(legacyscheme.Scheme, v1.EventSource{Component: componentKubelet, Host: string(nodeName)})
 	eventBroadcaster.StartLogging(klog.V(3).Infof)
 	if kubeDeps.EventClient != nil {
-		klog.V(4).Infof("Sending events to api server.")
+		klog.V(4).Info("Sending events to api server.")
```
Comment (draveness) [inline cmd/kubelet/app/server.go]: /lgtm
/ok-to-test
/priority backlog

**Review by mattjmcnaughton — APPROVED** [review]
/lgtm
/assign @derekwaynecarr

**Comment by mysunshine92** [comment]
cc @derekwaynecarr

**Comment by mysunshine92** [comment]
cc @derekwaynecarr
please add an approve lable,thanks.

**Comment by mysunshine92** [comment]
@yujuhong would you help to apporove this PR,thanks.

**Comment by spiffxp** [comment]
/close
This seems duped by a larger PR from the same author https://github.com/kubernetes/kubernetes/pull/79174

**Pull request closed without merging**